src/oop_ext/interface/_tests/test_interface.py

```diff
@@ -808,20 +808,3 @@ class Bar(Foo):
     mocker.patch.object(Foo, "foo", autospec=autospec)
 
     assert interface.IsImplementation(Bar, II) == (autospec or check_before)
-
-
-@pytest.mark.xfail(run=False, reason="core dumped")
-def testErrorOnInterfaceDeclaration(handled_exceptions):
-    def Check():
-        class Foo:
-
-            from oop_ext import interface
-
-            interface.ImplementsInterface(_InterfM1)
-
-    from oop_ext.foundation.handle_exception import IgnoringHandleException
-
-    with IgnoringHandleException():
-        Check()
```

Comment:
There is another test that check this? Otherwise I think you should rewrite this expecting an exception here